.env.example

```
@@ -31,4 +31,8 @@ OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT="True"
 # E2E_DEFAULT_CSV="data/ecg_csv/ddd/01M_1.csv"
 # E2E_PAPER_URL="http://localhost:8001/papers"
 # E2E_TARGET_COL="label"
-# E2E_COMBINED_CSV="data/ecg_csv/ddd/_e2e_combined.csv"
\ No newline at end of file
+# E2E_COMBINED_CSV="data/ecg_csv/ddd/_e2e_combined.csv"
+
+# Attack server base URL — set to a distinct host so demo traces show an external
+# attacker. Cloud Run deployment used for the live demo:
+# BASE_URL="https://attack-server-443663191326.europe-west4.run.app"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)